From f2a5724a042ab3eb9ee97c67dd4d6e72aba10d46 Mon Sep 17 00:00:00 2001
From: Sultanxda <sultanxda@gmail.com>
Date: Sat, 15 Aug 2015 19:28:35 -0700
Subject: [PATCH] CamcorderProfile: Add 1440p camcorder profile

Add encoding constants (regular and timelapse) for 2.5k 1440p resolution.

[YumeMichi: Adjusted for Q]

Change-Id: If3f8e7fef14eb9a47b47e382974f34f93698d1ff
---
 media/java/android/media/CamcorderProfile.java | 15 +++++++++++++--
 1 file changed, 13 insertions(+), 2 deletions(-)

diff --git a/media/java/android/media/CamcorderProfile.java b/media/java/android/media/CamcorderProfile.java
index 963b1d1..1fb245b 100644
--- a/media/java/android/media/CamcorderProfile.java
+++ b/media/java/android/media/CamcorderProfile.java
@@ -121,9 +121,15 @@ public class CamcorderProfile
      */
     public static final int QUALITY_2K = 12;
 
+    /**
+     * Quality level corresponding to 1440p (2560 x 1440) resolution.
+     * @hide
+     */
+    public static final int QUALITY_1440P = 13;
+
     // Start and end of quality list
     private static final int QUALITY_LIST_START = QUALITY_LOW;
-    private static final int QUALITY_LIST_END = QUALITY_2K;
+    private static final int QUALITY_LIST_END = QUALITY_1440P;
 
     /**
      * Time lapse quality level corresponding to the lowest available resolution.
@@ -194,10 +200,15 @@ public class CamcorderProfile
      */
     public static final int QUALITY_TIME_LAPSE_2K = 1012;
 
+    /**
+     * Time lapse quality level corresponding to the 1440p (2560 x 1440) resolution.
+     * @hide
+     */
+    public static final int QUALITY_TIME_LAPSE_1440P = 1013;
 
     // Start and end of timelapse quality list
     private static final int QUALITY_TIME_LAPSE_LIST_START = QUALITY_TIME_LAPSE_LOW;
-    private static final int QUALITY_TIME_LAPSE_LIST_END = QUALITY_TIME_LAPSE_2K;
+    private static final int QUALITY_TIME_LAPSE_LIST_END = QUALITY_TIME_LAPSE_1440P;
 
     /**
      * High speed ( >= 100fps) quality level corresponding to the lowest available resolution.
-- 
2.7.4

